pkgs/development/haskell-modules/configuration-hackage2nix.yaml

```diff
@@ -43,7 +43,7 @@ core-packages:
   - ghcjs-base-0
 
 default-package-overrides:
-  # LTS Haskell 14.12
+  # LTS Haskell 14.13
```

Comment:
You don't actually need to regenerate the `default-package-overrides` part of this file.

In general, @peti will do this once a week a or so.